Pipfile

```diff
@@ -5,11 +5,11 @@ verify_ssl = true
 
 [dev-packages]
 pycodestyle = "*"
-v = {editable = true,version = "*"}
+#v = {editable = true,version = "*"}
```

Comment:
@nirgn975 what is this package???,
when I create the Pipfile.lock to the new Pipfile I get an err msg: AttributeError: 'NoneType' object has no attribute 'url'

when I hashed this package it work fine. I couldn't manage to see whats wrong

Reply:
1. We write here in [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet), so you can use the "```" (three backticks) to start a code block and end it with another three of them.
2. Why do you create a new `Pipfile.lock`? You don't need to create the file, we already have it, it's generated automatcally every time someone do install/uninstall with pipenv.
3. You don't "hashed" it, you "commented" it (the hash here is to write comments, so you turn this line into a comment and that's why the system is ignoring it now).